src/Components/Stats/Stats.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import style from './Stats.module.scss'
+
+const Stats = () => {
+  return (
+    <section>
+      <h2>Stats Page Coming Soon!</h2>
+      <section className={style.statsDesc}>
+        <p>Check back soon for personal stats so you can see the positive impact Som Timer is having on your productivity and mental health. Stats will include data such as a log of pom cycles taken, mood and engagement tracking, and what types of content work best for you!</p>
+      </section>
```

Comment:
Can we use an `<article>` tag for this instead?

Reply:
Changed! Good suggestion